OlinkAnalyze/vignettes/bridging_E3072toEHT.Rmd

```diff
@@ -0,0 +1,682 @@
+        products. Since the NPX values are calculated on the same
+        samples, it is expected that an increase in 1 NPX in one product
+        would correspond to an increase of 1 NPX in the other product.
+        If the ranges are not similar, this suggests that 1 NPX is not
+        equivalent across products. By default, the cutoff is set to a
+        difference of less than 1 NPX between products.
+-   Are there sufficient counts in both products?
+    -   **Assessing if there are sufficient counts:** An assay's
+        absolute level of counts is important to consider as the
+        instruments used to generate NPX values have an inherent noise
+        level. To determine if there are sufficient counts in an assay
+        for bridging, the median number of counts in both products is
+        calculated, excluding data points with less than 10 counts. If
+        the median number of counts is less than the cutoff then the
+        assay does not have sufficient counts to be used for bridging.
+        The default cutoff is set to 150 counts, which is based on the
+        count quality control metrics for Explore products.
+
+For assays that are bridgeable, the shape of the NPX distribution is
+compared between the two products:
+
+-   **Assessing similarity of NPX distribution across products:** If the
+    three criteria outlined above are met then the assay is considered
+    bridgeable. Otherwise, bridging is not recommended for that assay.
+    If an assay is bridgeable, the similarity of the NPX distribution is
+    used to determine which method is recommended for bridging. The
+    Kolmogorov-Smirnov test, or KS test, is used to assess the
+    similarity of two distributions by calculating the KS statistic,
+    which is based on the empirical cumulative distribution function
+    (ECDF). Counts below 10 are excluded and the largest difference seen
+    in the ECDF becomes the KS statistic. If the KS statistic is above
+    the cutoff the distributions are considered to have different
+    shapes. In this case, a median shift is not sufficient to normalize
+    the data and quantile smoothing is recommended. If the distance is
+    less than the cutoff, then normalization using the median of paired
+    differences is recommended. By default this cutoff difference is set
+    to 0.2.
+
+An overview of these criteria is visualized below.
+
+```{r, fig.cap= fcap, eval=TRUE, echo=FALSE, out.width="100%"}
+knitr::include_graphics(normalizePath("../man/figures/assay_bridgeability.png"), error = FALSE)
+fcap <- "Criteria to determine the bridging recommendation for an assay. The assessment of linearity ensures bridging between signal in both platforms or noise in both platforms (but not between signal and noise. Similar NPX ranges and sufficient counts provide additional insight into an assays bridgeability. Distribution shape is assessed to determine recommended bridging method."
+```
+
```

Comment:
Optional, but is there a way to add some more space between the caption and the text below? Given the length of the caption and same formatting as the regular text, at first glance it looks like the caption is a continuation of the text normal text.

Reply:
In line 405, there is a missing closing parenthesis after "(but not between signal and noise"

Reply:
In line 405, change "assays" to "assay's"